.buildkite/scripts/health-report-tests/tests/slow-start.yaml

```diff
@@ -0,0 +1,17 @@
+- name: "Slow start pipeline"
+- config:
+  - pipeline.id: slow-start-pp
+    config.string: |
+      input { heartbeat {} }
+      filter { failure_injector { degrade_at => [register] } }
+      output { stdout {} }
+- expectation:
```

Comment:
This is an array containing three items, each of which is a map.
I think you need a map rather than an array of map.
```yaml
name: "Slow start pipeline"
config:
  - pipeline.id: slow-start-pp
    config.string: |
      input { heartbeat {} }
      filter { failure_injector { degrade_at => [register] } }
      output { stdout {} }
expectation:
  - status: yellow
  - symptom: "The pipeline is degraded or at risk of becoming unhealthy; 1 area is impacted and 1 diagnosis is available."
  - diagnosis:
      - cause: "pipeline is loading"
  - impacts:
    - description: "pipeline is loading"
    - impact_areas: "pipeline_execution"
  - details:
    - run_state: "LOADING"
```

Reply:
Yup, thank you for suggestion, makes sense to simplify. I have updated the change with [the recent commit](https://github.com/elastic/logstash/pull/16467/commits/5520880179741a31b19d38ae546342616d7ba2ba).